src/app/services/quiz.ts

```diff
@@ -353,7 +353,7 @@ export function convertAttemptToQuiz(attempt: QuizAttemptDTO): DisplayableQuiz |
         completedDate: attempt.completedDate,
         attempt: attempt,
         quizFeedbackMode: attempt.feedbackMode,
-        link: attempt.completedDate ? `/test/attempt/${attempt.id}/feedback` : `/test/attempt/${attempt.id}`,
+        link: attempt.completedDate ? `/test/attempt/${attempt.quizId}/feedback` : `/test/attempt/${attempt.quizId}`,
```

Comment:
This fixes the links for in progress tests, but the feedback links still seem to need the attempt id rather than the quiz id
```suggestion
        link: attempt.completedDate ? `/test/attempt/${attempt.id}/feedback` : `/test/attempt/${attempt.quizId}`,
```

Reply:
Ah, this is only the case for practice tests...